smartrouter/scripts/smartrouter/install

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+DISTRIBUTION_ZIP=jboss-bpmsuite-7.0.0.Alpha01-engine.zip
+DISTRIBUTION_ARTIFACT=kie-server-router-proxy-7.1.0.Beta3-redhat-1.jar
+DISTRIBUTION_ROUTER_PATH=jboss-bpmsuite-7.0-engine/${DISTRIBUTION_ARTIFACT}
+
+START_SCRIPT_NAME=start.sh
+SOURCES_DIR=/tmp/artifacts
+ROUTER_DIR=/opt/smartrouter
+
+unzip -q ${SOURCES_DIR}/${DISTRIBUTION_ZIP} -d ${SOURCES_DIR}
+mkdir -p ${ROUTER_DIR}
+cp -f ${SOURCES_DIR}/${DISTRIBUTION_ROUTER_PATH} ${ROUTER_DIR}
+
+echo -e '#!/bin/bash\n${JAVA_HOME}/bin/java' \
+        '-Dorg.kie.server.router.host=${KIE_ROUTER_HOST}' \
+        '-Dorg.kie.server.router.port=${KIE_ROUTER_PORT}' \
+        -jar ${ROUTER_DIR}/${DISTRIBUTION_ARTIFACT} > ${ROUTER_DIR}/${START_SCRIPT_NAME}
```

Comment:
No, CCT is not used in standalone images. I was more thinking about creating a simple `start.sh` file with all the information upfront, then just add it to the image and place it in correct location, instead of generating the script at build time.